cluster-scope/overlays/nerc-ocp-prod/ingresscontrollers/external-apps.ingress-controller.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: operator.openshift.io/v1
```

Comment:
This ingress controller needs to be configured to listen on a public address. That means we're first going to need to configure an appropriate interface on all the worker nodes (so probably a `NodeNetworkConfigurationPolicy` resource), and then we're going to need to perform some work -- of which I don't know the specifics -- to bind the ingress controller to the public address.